### ISSUE TITLE
[Vanilla bug] Add transparency for armour on armour stands

### ISSUE THREAD

**Issue by MushirMickeyJoe** [sp614x/optifine#2730]
This is different from issue [#863](https://github.com/sp614x/optifine/issues/863).

![image](https://user-images.githubusercontent.com/31923214/62504375-7721ff80-b7f8-11e9-8eed-7050a91274b9.png)

I wanted to add a transparency for a helmet texture I made, which works fine when worn, but fsr armour stands don't render transparency. Would be great if OF could fix this bug. Cheers!

**Comment by TheArcaneBrony** [comment]
Why don't you report this on the minecraft bug tracker instead?

**Comment by realjackasterisk** [comment]
If it really is a vanilla bug, I'm sure it's already been reported. Mojang's issue tracker has like a million posts, so trying to get a mod to do it is usually just faster.

**Comment by TheArcaneBrony** [comment]
Good point, it's interesting though that minecraft would render armor differently on armor stands then on players

**Comment by sp614x** [comment]
Probably relevant:
https://bugs.mojang.com/browse/MC-98308
https://bugs.mojang.com/browse/MC-109587
https://feedback.minecraft.net/hc/en-us/community/posts/360041568291-Allow-Better-Transparency-Support-for-Armor-in-Resource-Packs
